src/components/ConditionalFilter/API/Handler.ts

```diff
@@ -81,6 +87,30 @@ export const createCustomerOptionsFromAPI = (
   );
 };
 
+export const createAttributeProductVariantOptionsFromAPI = (
+  data: Array<{
+    node: {
+      name: string | null;
+      id: string;
+      slug?: string;
+      originalSlug?: string | null;
+      product?: {
+        name: string;
+      };
+    };
+  }>,
+): ItemOption[] =>
+  data.map(
+    ({ node }) =>
+      ({
+        // AttributeValue.name for product variant reference attributes
+        label: node.product ? `${node.product.name}: ${node.name}` : node.name ?? "",
```

Comment:
This matches our search vector in Saleor which means that if user has product variant "M" on produdct "T-shirt", name will be "T-shirt: M".

Entering the same input into combobox will also return this option